src/bp/ui-studio/src/web/views/FlowBuilder/diagram/index.jsx

```diff
@@ -94,6 +123,11 @@ export default class FlowBuilder extends Component {
       this.activeModel.setOffsetX(-_.min(nodes.map(({ x }) => x)) + PADDING)
       this.activeModel.setOffsetY(-_.min(nodes.map(({ y }) => y)) + PADDING)
 
+      // TODO: find the dark formula to center the selected element ...
+      if (centeredElement) {
```

Comment:
see two lines above for the dark formula ;)

Reply:
The lines above sets the canvas size & zoom level so all nodes are displayed, I tried a couple of things to center the desired node, but couldn't get the expected result